package.json

```diff
@@ -32,6 +32,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
+    "test:integration": "jest ./src/integration/pages/**/*.test.js --watch",
```

Comment:
Is this `--watch` accidental? Both commands seem to watch at present.

Reply:
I'm curious how explicitly specifying the directory here interacts with https://github.com/bbc/simorgh/pull/5764/files#diff-2d0cd5d10b9604941c38c6aac608178aR27. Are both necessary?

Reply:
Oops that's an accident with the watch task. I'll fix that.

`npm run test:integration` will run only the integration tests. If I did `npm run jest` it will run all jest tests, unit and integration. The new `jest.config` changes are essential to run integration tests against the new custom environment and to ensure the unit tests do not run in this environment. The line you pointed to is for matching unit test files to run. The line below excludes the integration folder when matching. The line for matching integration tests to run is here https://github.com/bbc/simorgh/pull/5764/files#diff-2d0cd5d10b9604941c38c6aac608178aR55

Confusing but I hope that gives some clarity

Reply:
That helps slightly, though I still find it a bit strange that the directory prefix (`src/integration`) needs to be specified in both places, and that [one pattern](https://github.com/bbc/simorgh/pull/5764/files#diff-2d0cd5d10b9604941c38c6aac608178aR56) matches both `.js` and `.jsx` whereas [the other](https://github.com/bbc/simorgh/pull/5764/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R35) only matches .js.